diagnostic.md

```diff
@@ -32,7 +32,20 @@ format described in the instructions). The page should have a title in the head
 and an unordered list (with three items) inside the body.
 
 <!-- your answer starts here -->
-
+```HTML
+<html>
+  <head>
+    <title>My Diagnostic web page</title>
+  </head>
+    <body>
+      <ul>
+        <li>Item 1
+        <li>Item 2
+        <li>Item 3
+      </ul>
+    </body>
+</html>
+```
```

Comment:
We were told in class they were not necessary. They were not missed, they were intentionally left off.